here-naksha-lib-handlers/src/main/java/com/here/naksha/lib/handlers/IntHandlerForEventHandlers.java

```diff
@@ -57,4 +66,22 @@ public IntHandlerForEventHandlers(final @NotNull INaksha hub) {
       return notImplemented(event);
     }
   }
+
+  private @NotNull Result validateWriteRequest(final @NotNull WriteRequest<?, ?> wr) {
+    Result result = null;
+    for (final WriteOp<?> wOp : wr.queries) {
+      final EventHandler eventHandler = (EventHandler) wOp.feature;
+      // Common Plugin specific validations
+      result = validateWritePluginRequest(eventHandler);
+      if (result instanceof ErrorResult) return result;
+      // TODO : handler specific validations in future, as needed
+      if (!eventHandler.getClassName().equals("com.here.naksha.lib.handlers.DefaultStorageHandler")) {
```

Comment:
@gunplar Just noticed, we shouldn't be using NOT (!) condition. We want to check storageId only if classname matches. So, it should be positive condition, and not negative.

Reply:
ah my bad, thanks for catching this